### PR TITLE
Additions to codegen initiative

### PR DESCRIPTION
**Pull request by erikjee**

### CODE REVIEW

File: content/initiatives/codegen.md [hunk 1]
```diff
@@ -34,9 +34,11 @@ We want to acknowledge the [work done in the Zig ecosystem](https://github.com/z
 
 ### What's next
 
-A significant coordinated effort is needed to reach the goal of making Rust faster than C. 
+The improvements the RFC gives are niche. What it does show is that there is room for improvement. While outperforming C by 5% might be unlikely, even a shift from -2% to +2% would significantly change Rust's story and, more importantly, its impact.
```
Comment (folkertdev) [inline content/initiatives/codegen.md:37]:
```suggestion
The improvements of RFC 3720 are only relevant for specific programs, but it shows that there is room for improvement. While outperforming C by 5% might be unlikely, even a shift from -2% to +2% would significantly improve Rust's story and, more importantly, its impact.
```

File: content/initiatives/codegen.md [hunk 2]
```diff
@@ -34,9 +34,11 @@ We want to acknowledge the [work done in the Zig ecosystem](https://github.com/z
...
 
-We are working on a plan and seeking funding to support this initiative. Have a look at our [preliminary workplan](/initiatives/workplans/codegen/) or contact us via [rustfasterthanc@trifectatech.org](mailto:rustfasterthanc@trifectatech.org).
+These improvements are achievable if we can better use Rust's advantages over C, i.e., having access to more information about how the compiler can optimize.
```
Comment (folkertdev) [inline content/initiatives/codegen.md:39]:
```suggestion
The gap with C can be closed if we can better use Rust's advantages over C, i.e., having access to more information about how the compiler can optimize.
```

Comment (folkertdev) [reply content/initiatives/codegen.md:39]: still not a great sentence, it doesn't say much.

Comment (erikjee) [reply content/initiatives/codegen.md:39]: Agreed, but it does hint at (that we understand) why this is even feasible

**Pull request merged**